core/src/main/java/com/aem/demo/core/filters/LoggingFilter.java

```diff
@@ -33,25 +33,18 @@
 /**
  * Simple servlet filter component that logs incoming requests.
  */
-@Component(service = Filter.class,
-           property = {
-                   Constants.SERVICE_DESCRIPTION + "=Demo to filter incoming requests",
-                   EngineConstants.SLING_FILTER_SCOPE + "=" + EngineConstants.FILTER_SCOPE_REQUEST,
-                   Constants.SERVICE_RANKING + ":Integer=-700"
+@Component(service = Filter.class, property = {Constants.SERVICE_DESCRIPTION + "=Demo to filter incoming requests", EngineConstants.SLING_FILTER_SCOPE + "=" + EngineConstants.FILTER_SCOPE_REQUEST, Constants.SERVICE_RANKING + ":Integer=-700"
```

Comment:
(cosmetic) Please don't exceed the maximal line length marked with a verical line in IDEA. Split long lines. For an annotation, the best way of splitting is between params, like 
```
@Component(
    service = Filter,class,
    property = {
        Constants.SOME + "=first_property",
        Constants.ANOTHEr + "=second_property"
    })
public class MyClass {... }
```
..and so on